src/components/ui/DateInput/DateInput.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen } from '@testing-library/react';
+import DateInput from './DateInput';
+
+describe('DateInput', () => {
+  const props = {
+    onChange: () => {},
+    dateValues: {
+      day: '30',
+      month: '02',
+      year: '2000',
+    },
+  };
+
+  it('renders with given class', () => {
+    render(<DateInput {...props} className="foo" />);
+
+    const content = screen.getAllByTestId('left-border-container')[0];
```

Comment:
There should be only one LeftBorderContainer, shouldn't it?

Reply:
Not really, I think they are nested here, thats why I have to use it like this

Reply:
Yeah you're right, it's nested inside of `Input`, ok